### PR TITLE
[PyOV] add function which create unique names for test files

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - create unique names for files which are used in python api tests

### Tickets:
 - 95935

### CODE REVIEW

**Review by p-wysocki — APPROVED** [review]

File: src/bindings/python/tests/test_utils/test_utils.py [hunk 1]
```diff
@@ -79,3 +80,9 @@ def deprecated_function4():
         deprecated_function3()
     with pytest.warns(DeprecationWarning, match="deprecated_function4 is deprecated and will be removed in version 2025.4. Use another function instead"):
         deprecated_function4()
+
+
+def create_filename_for_test(test_name):
+    python_version = str(sys.version_info.major) + "." + str(sys.version_info.minor)
+    filename = "./" + test_name[5:] + "_" + python_version
```
Comment (jiwaszki) [inline src/bindings/python/tests/test_utils/test_utils.py:87]:
```suggestion
    filename = "./" + test_name.replace("test", "") + "_" + python_version
```
Might be a littler cleaner solution, just an improvement.

File: src/bindings/python/tests/test_graph/test_manager.py [hunk 2]
```diff
@@ -38,10 +39,9 @@ def test_constant_folding():
     assert np.allclose(values_out, values_expected)
 
 
-def test_serialize_seperate_paths_kwargs():
+def test_serialize_seperate_paths_kwargs(request):
```
Comment (jiwaszki) [inline src/bindings/python/tests/test_graph/test_manager.py:42]: Where does `request` come from? Is it working locally?

File: src/bindings/python/tests/test_transformations/test_graph_rewrite.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 from openvino.runtime import opset8
 from openvino.runtime.passes import Manager, GraphRewrite, MatcherPass, WrapType, Matcher
 
-from utils.utils import count_ops, get_test_model, PatternReplacement
+from .utils.utils import count_ops, get_test_model, PatternReplacement
```
Comment (jiwaszki) [inline src/bindings/python/tests/test_transformations/test_graph_rewrite.py:7]:
```suggestion
from .utils.utils import count_ops, get_test_model, PatternReplacement
```
Why change all `utils` to relative `.utils`?

File: src/bindings/python/tests/test_utils/test_utils.py (hunk 1 above)
Comment (akuporos) [reply src/bindings/python/tests/test_utils/test_utils.py]: done

File: src/bindings/python/tests/test_transformations/test_graph_rewrite.py (hunk 3 above)
Comment (akuporos) [reply src/bindings/python/tests/test_transformations/test_graph_rewrite.py:7]: I added `__init__.py` file into test_transformations to find `test_utils`.
it caused to change the way how utils is imported.

**Review by jiwaszki — APPROVED** [review]

**Pull request merged**